src/main/resources/hudson/plugins/active_directory/ActiveDirectorySecurityRealm/help-cacheTtl.html

```diff
@@ -0,0 +1,8 @@
+<div>
+    Some LDAP servers may be slow, or rate limit client requests. In such cases enabling caching may improve performance
+    of Jenkins with the risk of delayed propagation of user changes from LDAP and increased memory usage on the master.
+    <br/>
+    TTL will determinate the time the information will be cached
+    <br/>
+    <b>Note:</b> The default configuration is to have TTL to 10 minutes
```

Comment:
what units is this measured in?  seconds, minutes, hours, days, average lifespan of an adult mayfly?

The dafult may be 10 minutes - but once you have changed this to something other than 10 you have no idea anymore